### PR TITLE
Feature/tests apt

### PR DESCRIPTION
**Pull request by guerinoni**
Add the first part of tests for apt APIT

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -3,17 +3,26 @@ name: tests
...
+      - name: run docker tests
+        run: ./scripts/docker-tests.sh
 
-      - name: run functional tests
-        run: ./scripts/functional-tests.sh
\ No newline at end of file
+  apt-tests:
+    name: apt-tests
+    runs-on: ubuntu-latest
+    container: guerra1994/go-mosquitto-ubuntu-env
```
Comment (rsora) [inline .github/workflows/tests.yml:21]: Can you please add the Dockerfile for this Image?

**Review by rsora — CHANGES_REQUESTED** [review]

**Review by rsora — APPROVED** [review]

File: .github/workflows/tests.yml (hunk 1 above)
Comment (guerinoni) [reply .github/workflows/tests.yml:21]: @rsora 
I added this Dockerfile but there is also another image and is not a simple docker file. For now, docker images are hosted on my GH https://github.com/guerinoni/go-mqtt-docker-env and https://github.com/guerinoni/go-mosquitto-ubuntu-env, if you want we can move this repo in `arduino` GH or add it as submodules
As you prefer...

File: handlers_apt_repositories_test.go [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+	}
+	defer s.mqttClient.Disconnect(100)
+
+	subscribeTopic(s.mqttClient, "0", "/apt/repos/list/post", s, s.AptRepositoryListEvent, false)
+
+	resp := ui.MqttSendAndReceiveTimeout(t, "/apt/repos/list", "{}", 1*time.Second)
+
+	if resp == "" {
+		t.Error("response is empty")
+	}
```
Comment (Pippolo84) [inline handlers_apt_repositories_test.go:33]:
```suggestion
	assert.NotEmpty(t, resp)
```

**Review by Pippolo84 — CHANGES_REQUESTED** [review]
Please fix that minor style issue

File: .github/workflows/tests.yml (hunk 1 above)
Comment (rsora) [reply .github/workflows/tests.yml:21]: Please add them under the Arduino org, documenting, and providing links in the readme in a proper section. :+1:

**Review by Pippolo84 — APPROVED** [review]

**Pull request merged**